web/modules/custom/rdf_entity/src/Form/OverviewRdf.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\taxonomy\Form\OverviewTerms.
+ */
+
+namespace Drupal\rdf_entity\Form;
+
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides terms overview form for a taxonomy vocabulary.
+ */
+class OverviewRdf extends FormBase {
```

Comment:
It looks like we can throw away this class. It was initially copied from the Taxonomy module, but it doesn't seem to do anything useful.